plugin/storage/kafka/options.go

```diff
@@ -69,12 +87,57 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultEncoding,
 		fmt.Sprintf(`(experimental) Encoding of spans ("%s" or "%s") sent to kafka.`, EncodingJSON, EncodingProto),
 	)
+	flagSet.String(
```

Comment:
Instead of repeating this over, could we define it only in one place e.g. `pkg/kafka/auth`?